packages/integrations/cloudflare/src/index.ts

```diff
@@ -65,21 +65,16 @@ export default function createIntegration(args?: Options): AstroIntegration {
 							(vite.resolve.alias as Record<string, string>)[alias.find] = alias.replacement;
 						}
 					}
-
-					vite.ssr = {
-						...vite.ssr,
-						target: 'webworker',
-					};
+					vite.ssr = vite.ssr || {};
+					vite.ssr.target = vite.ssr.target || 'webworker';
```

Comment:
Is this depending on the Solid integration's config running first, therefore not overriding that choice?

Reply:
yes, this i taking a user or integration config and therefore only setting best guess defaults, but respecting if another settings is overwriting those.